src/OrchardCore/OrchardCore.Recipes.Core/Services/RecipeExecutor.cs

```diff
@@ -55,79 +54,76 @@ public async Task<string> ExecuteAsync(string executionId, RecipeDescriptor reci
 
                 await using (var stream = recipeDescriptor.RecipeFileInfo.CreateReadStream())
                 {
```

Comment:
We need to check the history of this file. We may have changed it to read the document piece by piece for perf reasons.